### ISSUE TITLE
The runtime parameter of python3.7 is no longer supported

### ISSUE THREAD

**Issue by clementrey-dev** [aws-samples/sagemaker-ssh-helper#64]
Hello,

I am trying to use the project for [Forwarding TCP ports over SSH tunnel](https://github.com/aws-samples/sagemaker-ssh-helper#port-forwarding).

I am trying to deploy the section [IAM SSM Setup](https://github.com/aws-samples/sagemaker-ssh-helper/blob/main/IAM_SSM_Setup.md)

While running the command:
```bash
cdk -a "$APP" deploy SSH-IAM-SSM-Stack \
  -c sagemaker_role="$SAGEMAKER_ROLE_ARN" \
  -c user_role="$USER_ROLE_ARN"
```

I am getting the following error L:
```
6:02:16 PM | CREATE_FAILED        | AWS::Lambda::Function   | TrustRelationshipLambdaTrigger562CEAC4
Resource handler returned message: "The runtime parameter of python3.7 is no longer supported for creating or updating AWS Lambda functions. We recommend you use
the new runtime (python3.12) while creating or updating functions. 
```

Is there a workaround to avoid this issue ? 

Regards
Clement

**Comment by ivan-khvostishkov** [comment]
Hi, @clementrey-dev , thank you for pointing this out. It will be fixed in the new release, I'm working on it now.

**Comment by mfatihaktas** [comment]
Hi @ivan-khvostishkov, I also got blocked by this issue. Is there any update on the fix?
Is the only alternative the manual steps given [here](https://github.com/aws-samples/sagemaker-ssh-helper/blob/main/IAM_SSM_Setup.md#manual-setup)?

**Comment by mfatihaktas** [comment]
@ivan-khvostishkov I followed the manual steps given in the [doc](https://github.com/aws-samples/sagemaker-ssh-helper/blob/main/IAM_SSM_Setup.md#manual-setup), but unfortunately it did not work for me as far as I could tell. I am not able to see the sagemaker instances on SSM console under `Systems Manager/Fleet Manager/Managed nodes`. I am also not able to see the instances with
```
➜ sm-ssh list
SageMaker SSH Helper v2.2.0
Listing SageMaker instances for sagemaker
  Region: us-west-2
  Type: all
  FQDN: sagemaker
```
Neither with
```
➜ aws sagemaker list-notebook-instances
{
    "NotebookInstances": []
}
```

Can you please confirm that the manual steps given in the doc are up-to-date and valid?

**Comment by 108krohan** [comment]
Hi team, I produced a fix for the automated CDK + Cloud9 way. Request to please review once you have a chance to do so.

Thanks team,
Rohan

**Comment by 108krohan** [comment]
Hi @ivan-khvostishkov can you please look into the PR please? it's been open for two weeks so gentle reminder.

**Comment by ivan-khvostishkov** [comment]
Fixed and released in v2.2.1
